webaverse/app
# Add cryptoavatars

## Describe your changes
- Crypto avatars list added to the character select box.
- Character select preview box fixed to the bottom so when you scroll through all the lists it does not scroll with it.
- Changed the show arrow on hover for selecting the characters to just css and removed updateArrowPosition to decrease FPS ( No need to run all those functions for just 1 arrow ).
- Fixed the issue where some characters were facing the other way on preview.
- Fixed the app crashing when certain characters were selected ( problem was that not all models have the Head mesh called exactly "Head" so we modified the code a bit. )
- Styled the select boxes for the CryptoAvatar menu to fit the design better.
## What are the steps for a QA tester to test this pull request?
- Load the scene
- Open the character select menu
- And surf through the newly added CryptoAvatar characters
- Use pagination, hover over the characters and see them in the preview box
- Select the characters and playtest.
## Issue ticket number and link

## Screenshots and/or video

https://user-images.githubusercontent.com/55106546/183263052-12afeccf-24db-45a5-b3d6-e2638aeccb66.mov

## Checklist before requesting a review
